### PR TITLE
Fix calculation of inverted matrix

### PR DESCRIPTION
**Pull request by kaminova**
## What does the pull request do?
Fix calculation of the inverted matrix.

## What is the current behavior?
After 3D Transformation implementation transforms were broken. For example, if the transform is applied, pointer events are not raised.  Repro:
```
<Panel Width="100" Height="100" Background="Red" RenderTransform="rotate(45deg)" PointerPressed="OnPointerPressed" />

private void OnPointerPressed(object? sender, PointerPressedEventArgs e)
{
    //this is not called
    ...
}
```
      
## What is the updated/expected behavior with this PR?
This PR fixes transforms calculations.

### CODE REVIEW

**Comment by Takoooooo** [comment]
May you add a test for that case?

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0020949-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Comment by MarchingCube** [comment]
image

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0020994-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

File: tests/Avalonia.Base.UnitTests/Media/MatrixTests.cs [hunk 1]
```diff
@@ -30,6 +30,16 @@ public void Identity_Has_Inverse()
             Assert.True(matrix.HasInverse);
         }
 
+        [Fact]
+        public void Invert_Should_Work()
```
Comment (Gillibald) [inline tests/Avalonia.Base.UnitTests/Media/MatrixTests.cs:34]: I think the correct test should be A * A-¹ = I

Comment (jp2masa) [reply tests/Avalonia.Base.UnitTests/Media/MatrixTests.cs:34]: > I think the correct test should be A-¹ * A = I

I agree with this, although the way it's currently written follows the unit test concept a bit better as it doesn't depend on other functions.

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0021000-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Review by maxkatz6 — APPROVED** [review]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0021034-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Pull request merged**